tsjs.js

```diff
@@ -33,12 +33,33 @@ try {
         fs.writeFileSync(tempTsConfigFile, JSON.stringify(tsConfigLint));
         tsConfigFile = tempTsConfigFile;
     }
+
+    const program = Linter.createProgram(tsConfigFile, ".");
+    const linter = new Linter({fix: true}, program);
+
+    const files = Linter.getFileNames(program);
+    files.forEach((file) => {
+        const fileContents = program.getSourceFile(file).getFullText();
+        const configuration = Configuration.findConfiguration(tempTsLintFile, file).results;
+        linter.lint(file, fileContents, configuration);
+    });
+
+    tsfmt.processFiles(files, {
+        replace: true,
+        tsconfig: true,
+        tsconfigFile: tsConfigFile,
+        tslint: true,
+        tslintFile: tempTsLintFile,
+        editorconfig: false,
+        vscode: false,
+        vscodeFile: null,
+        tsfmt: true,
+        tsfmtFile: tempTsfmtFile,
+    })
+        .then(() => console.log('done'));

```

Comment:
`Tsjs lint and formatting completed successfully. See additional logs above for details.`